stories/SideEffects/useSessionStorageValue.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSessionStorageValue } from '../../src';
+
+interface IExampleProps {
+  /**
+   * Default value to return in case key not presented in SessionStorage.
+   */
+  defaultValue: string;
+  /**
+   * SessionStorage key to manage.
+   */
+  key: string;
+  /**
+   * Subscribe to window's `storage` event.
+   */
+  handleStorageEvent: boolean;
+}
+
+export const Example: React.FC<IExampleProps> = ({
+  key = 'react-hookz-ls-test',
```

Comment:
Should this be `react-hookz-ss-test`? Not that it really matters.

Reply:
actually in should